packages/amplify-util-mock/src/utils/lambda/populate-cfn-params.ts

```diff
@@ -61,9 +61,15 @@ const getAmplifyMetaParams = (
         );
         print.warning('This attribute will be undefined in the mock environment until you run `amplify push`');
       }
-      if (overrideApiToLocal && attribute === 'GraphQLAPIEndpointOutput') {
-        val = `http://localhost:${MOCK_API_PORT}/graphql`;
+
+      if (overrideApiToLocal) {
+        if (attribute === 'GraphQLAPIEndpointOutput') {
+          val = `http://localhost:${MOCK_API_PORT}/graphql`;
+        } else if (attribute === 'GraphQLApiKeyOutput') {
+          val = 'da2-fakeApiId123456';
```

Comment:
If it's not too difficult to move this value to an exported const that is imported when needed that would be nice